test/test_utils.py

```diff
@@ -338,6 +339,37 @@ def test_unescape_html(self):
         # HTML5 entities
         self.assertEqual(unescapeHTML('&period;&apos;'), '.\'')
 
+    def test_get_referrer_url(self):
+        # No-Referrer
+        self.assertEqual(get_referrer_url("https://example.com/page", "https://mozilla.org", "no-referrer"), None)
+        # Origin
```

Comment:
1. The comments are redundant
2. There are only 3-4 URLs used for all the tests. Put them in variables